### PR TITLE
dbeaver/dbeaver#2256 splash screen fixed for mac sonoma

### PR DESCRIPTION
**Pull request by serjiokov**
Splash screen fixed for mac sonoma

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverSplashHandler.java [hunk 1]
```diff
@@ -85,7 +85,14 @@ private void initVisualization() {
             versionInfoSizeString = product.getProperty("versionInfoSize");
             versionInfoColorString = product.getProperty("versionInfoColor");
         }
-
+        /*
+         * Related to a bug in MacOS Sonoma
+         * (https://github.com/eclipse-platform/eclipse.platform.swt/issues/772)
+         */
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverSplashHandler.java:92]: 1. Check that it is MacOS.
2. Return splash screen in MacOS (see previous PR)

Comment (serge-rider) [reply plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverSplashHandler.java:92]: And why use absolute coordinates? What are these numbers and why they are hardcoded?

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverSplashHandler.java (hunk 1 above)
Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverSplashHandler.java:92]: 1. done
2. done
3. This is a property value for product in general all specified in abs coords
```
      <property name="startupMessageRect" value="323,270,242,25"/>
      <property name="startupProgressRect" value="323,255,242,10"/>
```

Comment (serge-rider) [reply plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverSplashHandler.java:92]: These coordinates may change at any moment.
You can create new properties like startupMessageRectMacOS

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverSplashHandler.java (hunk 1 above)
Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverSplashHandler.java:92]: Not sure if I get it right, these approach of "man on the middle", properties are modified specifically for Mac OS after general reading procedure line:81, initially properties specified in plugin.xml.

**Comment by serjiokov** [comment]
@serge-rider  one more thing, the splash image in bmp format looks little bit accurate, would we replace the current (.png) ?

**Pull request closed without merging**